### DISCUSSION TITLE
Find the context of a given commit

### DISCUSSION THREAD

**Discussion by DamienCassou** [magit/magit#4268] [Support] [answered]
Hi everyone, 

very often, I have a commit and I would like to see a log of commits around this one. Usually, this is because the commit was part of a branch and I would like to see the other commits of the branch. It is easy to get the log of the commit's ancestors but I don't know how to see the descendants of this commits.

Workaround: open the log for `origin/master`, ask for more history by typing `+` several times and search for the beginning of commit SHA.

**Comment by kyleam** [comment, marked as answer]
There's `magit-log-merged`, which depends on [git-when-merged][1] being installed.  I use it very often to go from a particular commit to a log of the merged topic.

related issue: #2990

[1]: https://github.com/mhagger/git-when-merged

**Reply by DamienCassou** [reply]
Awesome, thank you!
